layouts/partials/head.html

```diff
@@ -99,6 +99,10 @@
           success: function(res) {
             $.each(res.data, function(index, item) {
               var description=item['product'] + ' ' + item['version'] + ' cluster on ' + item['cloud'];
+              if (item['error'] !== "") {
```

Comment:
This should be the first condition in the loop. Put it prior to initializing `description`.